run_shell_ssm_document.tf

```diff
@@ -0,0 +1,25 @@
+# ------------------------------------------------------------------------------
+# Create an SSM Document that allows creation of and configures
+# parameters for SSM Session Manager sessions in this account.
+# ------------------------------------------------------------------------------
+resource "aws_ssm_document" "session_manager_preferences" {
+  content = jsonencode({
+    description = "Document to hold regional settings for Session Manager"
+    # For a full list of input values, see here:
+    # https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-schema.html
+    inputs = {
+      cloudWatchEncryptionEnabled = false
+      cloudWatchLogGroupName      = aws_cloudwatch_log_group.ssm_sessions.name
+      cloudWatchStreamingEnabled  = true
+      shellProfile = {
+        linux   = var.linux_shell_profile
+        windows = var.windows_shell_profile
+      }
+    }
+    schemaVersion = "1.0"
```

Comment:
Is there a particular reason we're using an old schema version? The [documentation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ssm_document) for this resource type warns that you must destroy and recreate documents with a schema version <2.0 if you want to make changes.

Reply:
Ah, that was the piece I was missing. Thanks!